d3a_api_client/rest_market.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from d3a_api_client.rest_device import logging_decorator
+from concurrent.futures.thread import ThreadPoolExecutor
+from d3a_api_client.websocket_device import WebsocketMessageReceiver, WebsocketThread
+from d3a_api_client.utils import retrieve_jwt_key_from_server, post_request, get_request
+
+
+root_logger = logging.getLogger()
+root_logger.setLevel(logging.ERROR)
+
+
+class RestMarketClient:
+
+    def __init__(self, simulation_id, device_id, domain_name,
+                 websockets_domain_name):
+        self.simulation_id = simulation_id
+        self.device_id = device_id
+        self.domain_name = domain_name
+        self.jwt_token = retrieve_jwt_key_from_server(domain_name)
+
+        self.dispatcher = WebsocketMessageReceiver(self)
+        self.websocket_thread = WebsocketThread(simulation_id, device_id, self.jwt_token,
+                                                websockets_domain_name, self.dispatcher)
+        self.websocket_thread.start()
+        self.callback_thread = ThreadPoolExecutor(max_workers=5)
```

Comment:
May be an overkill but would it makes sense to set max_workers via constants ?

Reply:
Kind of, I'll do it though since you mentioned it